src/routing/routing.component.test.tsx

```diff
@@ -117,7 +120,9 @@ describe('Routing component', () => {
         <MemoryRouter
           initialEntries={[{ key: 'testKey', pathname: '/test_link' }]}
         >
-          <Routing />
+          <ThemeProvider theme={theme}>
+            <Routing />
+          </ThemeProvider>
```

Comment:
Sorry, one last change! Adding the `ThemeProvider` to the mounted wrapper has resulted in some massive additions to the snapshots that we just don't need. If you take it out from all the mounted wrappers in this file, you should get some more manageable snapshots. Once it's done, compare the new snapshots to what is currently on develop branch and as long as they look fairly similar in size, they're probably good to go!

Reply:
No worries! I didn't know it wasn't needed. I thought the `Routing` component would need it but turns out it doesn't. I don't think it should be removed though as we will have to rewrite it in `@testing-library` in the future which will actually require `ThemeProvider` since it renders the entire component out, and all the styling obviously requires the `theme` object to be in context.

Reply:
Personally, if we remove it, we can more easily test the snapshot is correct during the migration to testing-library, plus just leaving the import by itself will make the linter complain. But that's just my opinion